repos/fdbt-site/src/pages/api/searchOperators.ts

```diff
@@ -117,6 +133,19 @@ export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
         updateSessionAttribute(req, MULTIPLE_OPERATOR_ATTRIBUTE, { selectedOperators });
 
         if (continueButtonClick) {
+            const operatorsWithNoServices = await getOperatorsWithoutServices(selectedOperators);
+            if (operatorsWithNoServices.length > 0) {
+                errors.push({
+                    errorMessage: `Some of the selected operators have no services`,
```

Comment:
Could we slightly alter this error message, just to help advise the user what to do: maybe something like

`Some of the selected operators have no services. To continue you must only select operators which have services in BODS.`